draft-ietf-sedate-datetime-extended.md

```diff
@@ -262,14 +261,15 @@ Time Zone:
   to a timestamp, with the caveat that some local times may have zero or
   multiple possible timestamps due to nearby Daylight Saving Time
   changes or other changes to the UTC offset of that time zone.
-  Unlike the UTC offset of a timestamp which makes no claims about the
-  UTC offset of other related timestamps (and which is therefore
-  unsuitable for performing local-time operations such as "one day
-  later"), a time zone also defines how to derive new timestamps based
-  on differences in local time.
+  A time zone also defines how to derive new timestamps based on
+  differences in local dates and times.
   For example, to calculate "one day later than this timestamp in San
   Francisco", a time zone is required because the UTC offset of local
   time in San Francisco can change from one day to the next.
+  On the other hand, the UTC offset of a timestamp makes no claims
+  about the UTC offset of other related timestamps.
+  For this reason, UTC offsets are insufficient to perform local-time
+  operations like "one day later".
```

Comment:
Re-ordered this section to first explain what a time zone is used for (including an example), and only then explain how and why an offset can't do the same thing.  I think this order may be a clearer way to explain this complicated problem.